app/models/concerns/curator/controlled_terms/id_from_auth_findable.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Curator
+  module ControlledTerms
+    module IdFromAuthFindable
+      extend ActiveSupport::Concern
+
+      class_methods do
+        def find_id_from_auth(id_from_auth)
+          find_id_from_auth!(id_from_auth)
+        rescue ActiveRecord::RecordNotFound
+          nil
+        end
+
+        def find_id_from_auth!(id_from_auth)
+          jsonb_contains(id_from_auth: id_from_auth).first!
+        end
```

Comment:
Raise an error (`Multiple results found for #{id_from_auth}!`) if there are multiple results